pkg/config/config.go

```diff
@@ -595,6 +595,18 @@ func (c *Config) discoverRecursivePackages(ctx context.Context) error {
 			subPkgLog := pkgLog.With().Str("sub-package", subPkg).Logger()
 			subPkgCtx := subPkgLog.WithContext(pkgCtx)
 
+			if len(conf.Exclude) > 0 {
+				p := pathlib.NewPath(subPkg)
+				relativePath, err := p.RelativeToStr(pkgPath)
+				if err != nil {
+					return fmt.Errorf("failed to get path for %s relative to %s: %w", subPkg, pkgPath, err)
```

Comment:
Please change this to use the StackErr: https://github.com/vektra/mockery/blob/master/pkg/stackerr/stackerr.go#L21

Reply:
Done in 1407ff88ea428c24756ba122d01fc79392977b7e